src/SFA.DAS.Employer.PR.Web/Views/YourTrainingProviders/Index.cshtml

```diff
@@ -47,23 +48,23 @@
                     <th scope="col" class="govuk-table__header">Training provider</th>
                     <th scope="col" class="govuk-table__header">Permission to add apprentice records</th>
                     <th scope="col" class="govuk-table__header">Permission to recruit apprentices</th>
-                @if (@Model.IsOwner)
-                {
-                    <th scope="col" class="govuk-table__header">Change permissions</th>   
-                }
+                    @if (@Model.IsOwner)
+                    {
+                        <th scope="col" class="govuk-table__header">Change permissions</th>
+                    }
                 </tr>
                 </thead>
 
                 <tbody class="govuk-table__body">
-                @foreach (var permissionDetails in ale.Permissions.OrderBy(p=>p.ProviderName))
+                @foreach (var permissionDetails in ale.Permissions.OrderBy(p => p.ProviderName))
```

Comment:
All the ordering should be controlled in the model, so we can add unit tests around it